internal/test_geospatial_calculate_score.go

```diff
@@ -0,0 +1,61 @@
+package internal
+
+import (
+	"github.com/codecrafters-io/redis-tester/internal/data_structures/location"
+	"github.com/codecrafters-io/redis-tester/internal/instrumented_resp_connection"
+	"github.com/codecrafters-io/redis-tester/internal/redis_executable"
+	"github.com/codecrafters-io/redis-tester/internal/resp_assertions"
+	"github.com/codecrafters-io/redis-tester/internal/test_cases"
+	testerutils_random "github.com/codecrafters-io/tester-utils/random"
+	"github.com/codecrafters-io/tester-utils/test_case_harness"
+)
+
+func testGeospatialCalculateScore(stageHarness *test_case_harness.TestCaseHarness) error {
+	b := redis_executable.NewRedisExecutable(stageHarness)
+	if err := b.Run(); err != nil {
+		return err
+	}
+
+	logger := stageHarness.Logger
+
+	client, err := instrumented_resp_connection.NewFromAddr(logger, "localhost:6379", "client")
+	if err != nil {
+		logFriendlyError(logger, err)
+		return err
+	}
+	defer client.Close()
+
+	locationKey := testerutils_random.RandomWord()
+	locationSet := location.GenerateRandomLocationSet(testerutils_random.RandomInt(2, 4))
+	locations := locationSet.GetLocations()
+
+	// Add locations
+	for _, loc := range locations {
```

Comment:
```suggestion
	for _, location := range locations {
```

(Avoid abbreviations unless it's a struct method receiver)